src/redux/appointment/appointmentsSlice.js

```diff
@@ -0,0 +1,92 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const url = 'http://localhost:3001';
+
+const createAppointment = createAsyncThunk('user/createAppointment', async (data) => {
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${JSON.parse(localStorage.getItem('user'))?.token}`,
+      },
+      body: JSON.stringify(data),
+    });
```

Comment:
I encountered a 404 error when attempting to submit an appointment. It seems to be originating from the slice. Could you please investigate and resolve this issue?